### PR TITLE
Parse and report HTTP query params.

### PR DESCRIPTION
**Pull request by pglombardo**
This adds URL query reporting for the HTTP server instrumentation.

### CODE REVIEW

**Comment by pglombardo** [comment]
screen shot 2017-10-25 at 15 20 44

File: src/tracing/instrumentation/httpServer.js [hunk 1]
```diff
@@ -65,10 +66,12 @@ function requestListener(req, res) {
   }
 
   res.on('finish', function() {
+    var urlParts = url.parse(req.url);
```
Comment (bripkens) [inline src/tracing/instrumentation/httpServer.js]: I noticed before that this is a really expensive operation and removed it in some location within this module (also within other Instana components). Alternatively, could we look for the first `?` and grab everything after that till the end of the string?

**Comment by bripkens** [comment]
We should extend the test cases to test for params.

**Comment by bripkens** [comment]
Maybe I am wrong, but I believe to remember that we don't include the leading `?` in the other sensors, no?

File: src/tracing/instrumentation/httpServer.js (hunk 1 above)
Comment (pglombardo) [reply src/tracing/instrumentation/httpServer.js]: I was wondering why you didn't use it...

**Comment by pglombardo** [comment]
All good points.  I'll redo.  And add tests.

**Comment by pglombardo** [comment]
BTW coding this was a total 5 minutes.  Fighting the ecosystem that I'm not familiar with...1 hour+ ¯\_(ツ)_/¯

**Pull request closed without merging**